src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,27 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private Connection connection;
+
+    public boolean connect(String databaseName, String user, String password) {
```

Comment:
This method has redutant logic. According to the task this method should return true if the connection with database was successful and false otherwise, no messages required. This method could be in 2 times shorter.

Reply:
I agree with @Shtramak

Reply:
I agree too